src/lib/components/FrontPage.svelte

```diff
@@ -1,6 +1,9 @@
 <script lang="ts">
 	let inputfield: string = '';
 
+
+  let color:string;
```

Comment:
Hvad gør manglen på mellemrum mellem typen og identifying factor her?
```suggestion
  let color: string;
```

Reply:
ingenting

Reply:
okay tak